Gemfile.lock

```diff
@@ -94,13 +104,15 @@ GEM
     webrick (1.9.1)
 
 PLATFORMS
+  arm64-darwin-24
   x86_64-linux
 
 DEPENDENCIES
   asciidoctor-diagram (~> 2.3.2)
   jekyll (~> 4.4.1)
   jekyll-asciidoc (~> 3.0.1)
   jekyll-feed (~> 0.12)
+  jekyll-redirect-from
```

Comment:
I think we're missing a corresponding `Gemfile` change like:

```
diff --git a/Gemfile b/Gemfile
index 5423258..86df08d 100644
--- a/Gemfile
+++ b/Gemfile
@@ -16,4 +16,5 @@ group :jekyll_plugins do
   gem "jekyll-sass-converter", "~> 3.1.0"
   gem "jekyll-seo-tag", "~> 2.8.0"
   gem "jekyll-toc", "~> 0.19.0"
+  gem 'jekyll-redirect-from', '~> 0.16.0'
 end
```

Reply:
Oh yeah I was starting to think I didn't need the plugin. Just an incomplete revert.

Reply:
or was the plugin an aborted experiment, I guess the page generator is a separate thing?